srcDoc/release_notes.rst

```diff
@@ -51,6 +51,19 @@ Release Notes Master
 What's New in Master
 ~~~~~~~~~~~~~~~~~~~~
 
+Use default `UDPv4` + `SHMEM` as the default transport configuration (#80)
```

Comment:
I think I missed removing the first default

Use `UDPv4` + `SHMEM` as the default transport configuration (#80)

Reply:
Done